canonical/ubuntu.com
# Update react to v18

## Done
- Update React v17 to v18
- Update React Query v3 to v5
- Update React Components v0.38 to v0.59
- Update ESLint v7 to v9
- Many dependencies have been updated in this PR. See `package.json` for the updated dependencies.
- Fixed tests and code for react-query according to the updated version.

## QA
- Check out this feature branch or check the demo https://ubuntu-com-14108.demos.haus/
- Check all the pages to see if anything is broken
- Check all github checks have passed

## Issue / Card

Fixes #https://warthogs.atlassian.net/browse/WD-13589
